bedrock/firefox/templates/firefox/personal.html

```diff
@@ -0,0 +1,55 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+ {% extends "base-resp.html" %}
+
+{% block page_title_prefix %}{{_('Firefox Web browser')}} - {% endblock %}
+{% block page_title %}{{_('When it\'s personal, choose Firefox')}}{% endblock %}
+
+{% block page_desc %}Choose Firefox when you want to browse the Web without the Web browsing you.{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'personal' %}
+{% endblock %}
+
+{% block site_header_nav %}{% endblock %}
+{% block site_header_logo %}{% endblock %}
+
+{% block alt_header %}
+<div class="header-content">
+  <img src="{{ static('img/firefox/personal/logo.png') }}" width="147" alt="{{_('Firefox logo')}}" />
+  <h1>{{_('When it\'s personal,<span>choose Firefox</span>')}}</h1>
+  {{ download_firefox(force_direct=true, simple=true) }}
+</div>
+{% endblock alt_header %}
+
+{% block extrahead %}{% endblock %}
+
+{% block js_pretrack %}{% endblock %}
+
+{% block content %}
+<main role="main">
```

Comment:
@schalkneethling destination for these links is noted in the [copy document](https://docs.google.com/document/d/1QfQKx0DSOmcXIVPkA1nQ1TEWS5JbhIyZs0ZgsiyKXC4/edit).